platform/darwin/src/NSExpression+MGLAdditions.h

```diff
@@ -9,6 +9,54 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef NSString *MGLExpressionStyleFunction NS_STRING_ENUM;
+typedef NSString *MGLExpressionInterpolationMode NS_TYPED_EXTENSIBLE_ENUM;
+
+/**
+ An `NSString` identifying the `zoomLevel` operator in an `NSExpression`.
+ 
+ This attribute corresponds to the
+ <a href="https://www.mapbox.com/mapbox-gl-js/style-spec/#expressions-zoom"><code>zoom</code></a>
+ expression reference in the Mapbox Style Specification.
+ */
+extern MGL_EXPORT const MGLExpressionStyleFunction MGLExpressionStyleFunctionZoomLevel;
```

Comment:
These aren’t functions, they’re inputs to the expression, represented as variables. Rename `MGLExpressionStyleFunction` to `MGLExpressionVariableName`. Better yet, replace these constants with class properties on a category of NSExpression that return variable expressions. For example, the property `NSExpression.mgl_zoomLevelVariableExpression` would return an instance of NSExpression that was initialized using `+[NSExpression expressionForVariable:]` in a `dispatch_once` block.